src/platform/windows/window.rs

```diff
@@ -1019,13 +1023,22 @@ unsafe fn init(
     // making the window transparent
     if attributes.transparent && !pl_attribs.no_redirection_bitmap {
         let bb = dwmapi::DWM_BLURBEHIND {
-            dwFlags: 0x1, // FIXME: DWM_BB_ENABLE;
+            dwFlags: dwmapi::DWM_BB_ENABLE | dwmapi::DWM_BB_BLURREGION,
             fEnable: 1,
-            hRgnBlur: ptr::null_mut(),
+            hRgnBlur: CreateRectRgn(0, 0, -1, -1), // makes the window transparent
```

Comment:
Do we need to manually free this region using `DeleteObject`?

Reply:
Yep, good catch.